spec/requests/atmosphere/api/clew_spec.rb

```diff
@@ -172,20 +167,29 @@ def cs
         expect(clew_at_response['appliance_types'][1]['matched_flavor']).to clew_flavor_eq  flavor
 
         #TODO complete this test to check all the returned fields
-
       end
+    end
+
+    it 'does not return AT with visible_to owner when user is not owner' do
```

Comment:
I wonder, though - should 'owner-only' ATs really be hidden from admins? Doesn't sound very "adminlike" if you know what I mean... :)